template2.html

```diff
@@ -0,0 +1,180 @@
+                  A daily selection of privately personalized reads; no accounts or sign-ups required.
+                 </p>
+            </div>
+            <footer>
+               <div id="skills_1" class="skills">
+                 <p>html</p>
+                 <p>css</p>
+                 <p>javaScript</p>
+               </div>
+               <div class="external_link">
+                   <a href="#">See Project</a>
+               </div>
+              </footer>
+  
+            
+          </section>
+          <section class="p">
+            <div>
+                  <img  class="project_image" src="statics/images/Snapshoot_Portfolio2.png" alt="snapshoot image project">
+                  <h1>Tonic</h1>
+                  <div id="general_inf_2">
+                      <p class="title">CANOPY</p>
+                      <img class="counter" src="statics/images/Counter.png" alt="counter"/>
+                      <p class="job-title">Back End Dev</p>
+                      <img class="counter" src="statics/images/Counter.png" alt="counter">
+                      <p class="year">2015</p>
+                  </div>
+              
+                 <p class="project_description">
+                  A daily selection of privately personalized reads; no accounts or sign-ups required.
+                 </p>
+            </div>
+            <footer>
+               <div id="skills_2" class="skills">
+                 <p>html</p>
+                 <p>css</p>
+                 <p>javaScript</p>
+               </div>
+               <div class="external_link">
+                   <a href="#">See Project</a>
+               </div>
+              </footer>
+  
+            
+          </section>
+          <section class="p">
+            <div>
+                  <img  class="project_image" src="statics/images/Snapshoot_Portfolio3.png" alt="snapshoot image project">
+                  <h1>Multi-Post
+                    Stories</h1>
+                  <div id="general_inf_3">
+                      <p class="title">CANOPY</p>
+                      <img class="counter" src="statics/images/Counter.png" alt="counter"/>
+                      <p class="job-title">Back End Dev</p>
+                      <img class="counter" src="statics/images/Counter.png" alt="counter">
+                      <p class="year">2015</p>
+                  </div>
+              
+                 <p class="project_description">
+                  A daily selection of privately personalized reads; no accounts or sign-ups required.
+                 </p>
+            </div>
+            <footer>
+               <div id="skills_3" class="skills">
+                 <p>html</p>
+                 <p>css</p>
+                 <p>javaScript</p>
+               </div>
+               <div class="external_link">
+                   <a href="#">See Project</a>
+               </div>
+              </footer>
+  
+            
+          </section>
+          <section id="about_me">
```

Comment:
- [ ] In the Figma design this section has just a bit more of the left margin and it is more centered.

Yours: 
![image](https://user-images.githubusercontent.com/73256642/133896991-aa110138-7cdc-4d05-a84f-21a7a71df1ba.png)
Figma:
![image](https://user-images.githubusercontent.com/73256642/133896996-6d9c25dd-6d75-4946-bc15-bdda65b2c10a.png)